pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "wrroc-ga4gh-cloud-converter"
```

Comment:
Feel free to think of a better name. It doesn't have to have wrroc or ga4gh or cloud or converter in its name. It can be just any word that is easy to search (does not bring up 100 different things) and type and remember, maybe something in your language? Can also be an acronym for something (though I find most of them painful).

Anyway, this shouldn't block this PR - just think about it on the side and let's do the renaming once you found something. The project would obviously benefit from a better name.